src/_includes/episodes_layout.njk

```diff
@@ -22,8 +22,12 @@
           Published {{ publish_date | formatDate }} -
           <a href="{{ spotify_link }}" class="text-primary-100 dark:text-secondary-100 hover:underline">Listen on your favourite podcast player</a>
         </p>
+        <div class="aspect-w-16 aspect-h-9">
+          <iframe🤽‍♂️ id="youtubePlayer" width="100%" height="100%"
+            src="https://www.youtube.com/embed/{{ youtube_id }}?enablejsapi=1" title="YouTube video player" frameborder="0" 
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        </div>
       </div>
-      <p>{{ youtube_id | youtubeLink }}</p>
```

Comment:
we can probably remove the `youtubeLink` helper at this stage then... I don't think we are using it somewhere else.

Reply:
It's still used for YouTube comments link and on `index.njk`. May be removable in the future.